src/compiler/diagnosticMessages.json

```diff
@@ -2048,7 +2048,7 @@
         "category": "Error",
         "code": 2567
     },
-    "Type '{0}' is not an array type. Use compiler option '--downlevelIteration' to allow iterating of iterators.": {
+    "Type '{0}' is not an array type. Either use the '--downlevelIteration' compiler option to allow iterating on iterators, or set the '--target' option to 'es2015' or above.": {

```

Comment:
What about the below error message?

Reply:
Ok, will change. Thanks ;)

Reply:
?

Reply:
@DanielRosenwasser ?

Reply:
@DanielRosenwasser could you add to comment message which needs to use? Or point me out where you posted it?

Reply:
@DanielRosenwasser ?

Reply:
Sorry, for context, there's an error directly below:

```
Type '{0}' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.
```

Reply:
Ok, will change. Thanks

Reply:
@DanielRosenwasser fixed :+1: